subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/JvmVendorSpec.java

```diff
@@ -52,8 +52,10 @@ public abstract class JvmVendorSpec {
      * A constant for using <a href="https://developer.ibm.com/languages/java/semeru-runtimes/">IBM Semeru Runtimes</a> as the JVM vendor.
      *
      * @since 7.4
+     * @deprecated All IBM runtimes are Semeru runtimes, so no point in having two separate vendor definitions. IBM should be used instead.
      */
-    public static final JvmVendorSpec IBM_SEMERU = matching(KnownJvmVendor.IBM_SEMERU);
+    @Deprecated
```

Comment:
Please file an issue to add nagging in 8.1

Reply:
https://github.com/gradle/gradle/issues/23155